lib/storage.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+module.exports = exports = {};
+
+exports.createItem = function(schemaName, item) {
+  if (!schemaName) return Promise.reject(new Error('expected schema name'));
+  if (!item) return Promise.reject(new Error('expected item'));
+
+  let json = JSON.stringify(item);
+  return fs.writeFileProm(`${__dirname}/../data/${schemaName}/${item.id}.json`, json)
+  .then( () => item)
+  .catch( err => Promise.reject(err))
+};
+
+exports.fetchItem = function(schemaName, id) {
+  if (!schemaName) return Promise.reject(new Error('expected schema name'));
+  if (!id) return Promise.reject(new Error('expected id'));
+
+  return fs.readFileProm(`${__dirname}/../data/${schemaName}/${id}.json`)
+  .then(data => {
+    try {
+      let item = JSON.parse(data.toString());
+      return item;
+    } catch (err) {
+      return Promise.reject(err);
+    }
+  })
+  .catch(err => Promise.reject(err));
+};
```

Comment:
You are currently missing a delete route. If you wanted to add in a delete route you will need to do a few things. Here in your `storage.js` file you will need to include a method to remove data from the storage object (i.e. `exports.deleteItem`) and then you will need to call that inside your `oohdata-route.js` file.